src/liballoc/arc.rs

```diff
@@ -894,6 +895,13 @@ impl<T: ?Sized + Hash> Hash for Arc<T> {
     }
 }
 
+#[stable(feature = "rust1", since = "1.6.0")]
```

Comment:
Ah these attributes are fine, but they'll need a new feature name as `rust1` is already taken. Perhaps something like `from_for_ptrs` or something like that?